lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -176,6 +176,16 @@ namespace Js
             return FALSE;
         }
 
+        if (object->IsProtoImmutable())
+        {
+            // 19.1.3:
```

Comment:
Section numbers change between spec versions, so you should specify "ES2016" here.